### ISSUE TITLE
VoiceOver reads out multiple resistance values after sliders stop being adjusted

### ISSUE THREAD

**Issue by JRomero0613** [phetsims/resistance-in-a-wire#193]
Test device: MacBook Air

Operating System: macOS High Sierra v. 10.13.6

Browser: Safari 12.0.13606.2.11

Problem Description: For https://github.com/phetsims/QA/issues/219 when using VoiceOver and inputting many inputs to the slider the read outs are inconsistent with what they should be. When adjusting sliders using the mouse all that is read out is "as letter _ grows/shrinks letter R grows/shrinks", not the actual value of the slider that is being manipulated. This makes is so that without being able to visually see the slider value on the screen users will have no idea what the value of any sliders are. When using keyboard navigation to change slider values and inputting a lot of changes VoiceOver will read out the slider values as intended, but read out previous values of the resistance before reading out the current value. This makes it hard to connect the slider values and the resistance value without being able to visually see the values.

Steps to reproduce: 
**Mouse Bug:**
1. Open the sim in a new tab and turn on VoiceOver
2. Enter the web area by hitting control-option-shift-down arrow
3. Manipulate any slider values using the mouse, VoiceOver should read out the value of the resistance but no values of any sliders.

**Keynav Bug:**
1. Open the sim in a new tab and turn on VoiceOver
2. Navigate to a slider using tab 
3. Using the arrow keys input a lot of commands to change the slider values (don't use left and right in quick consecutive strokes as it will turn off quick-nav)
4. Wait for VoiceOver to read out the results, the slider value should be read out, followed by random resistance values that occurred during the input, and finally the resistance value at the current slider values.

**Comment by jessegreenberg** [comment]
For the mouse bug, this is the expected behavior. The "accessible" slider in the parallel DOM doesn't have focus during mouse interaction so we don't expect it to announce aria-valuetext updates. The same is true in john-travoltage.

For the Keynav bug, this is because VoiceOver reads all alerts that are queued up even during interaction (unlike JAWS and NVDA). So when using the arrow keys quickly from 0.01 ohms to 1 ohms I would expect to hear all of these alerts on VoiceOver from bottom to top
![capture](https://user-images.githubusercontent.com/6396244/48373221-32b16b00-e68f-11e8-9bec-f4295c20c0a6.PNG)

We could improve this but it would delay the release of this sim. This is been the behavior in VO for this sim for a long time, including the previous RCs. @emily-phet should this receive any work?

**Comment by emily-phet** [comment]
@jessegreenberg - is this the case for JT as well? So, have we just accepted this behavior for sliders in VO in the past? I can't recall...

**Comment by jessegreenberg** [comment]
Fix added above, pending testing in next RC.

**Comment by emily-phet** [comment]
@jessegreenberg Thanks for going ahead!

**Comment by jbphet** [comment]
The fix for this has been propagated to the 1.6 release branch.

**Comment by jessegreenberg** [comment]
Thanks @jbphet, removing my assignment then.

**Comment by lmulhall-phet** [comment]
@jessegreenberg, @jbphet, the keynav bug is fixed for the sliders, however during testing I noticed that the same thing can occur with the reset all button if you press and hold the return key. Seems like a weird use case to me, but I thought I'd mention it if implementing https://github.com/phetsims/resistance-in-a-wire/commit/ffb9e758203cdc631d115d3b231999b9a4dc1481 for the reset all button would be easy and/or anyone thinks something should be done about it.

**Comment by KatieWoe** [comment]
Above found during https://github.com/phetsims/QA/issues/239.  Adding to connect the two issues.

**Comment by emily-phet** [comment]
If this makes sense to @jessegreenberg, no need to elaborate...but for me it's not clear what the bug is for the reset all case. When reset all is selected, isn't the alert only something like "sim reset"? (I believe it's slightly longer than this, but ends with "sim reset".) Is the "sim reset" alert not read out?

**Comment by lmulhall-phet** [comment]
@emily-phet, pressing and holding the reset all button adds a ton of "sim reset" alerts to the alert queue, so each of them is read as they are dequeued.

**Comment by emily-phet** [comment]
Ahh - i see. thanks!

**Comment by jbphet** [comment]
I'm going to leave this to @jessegreenberg - let me know if changes are made and I can get them into the 1.6 release branch if needed.

**Comment by jessegreenberg** [comment]
The above commit is a workaround that fixes this issue for the Reset All button until we finish https://github.com/phetsims/sun/issues/424. @jbphet or @emily-phet can you please review https://github.com/phetsims/resistance-in-a-wire/issues/193#issuecomment-448378134? If that is OK then the above commit can be merged into the release branch. Otherwise we will need to get through https://github.com/phetsims/sun/issues/424 before this sim can be released.

**Comment by jessegreenberg** [comment]
Thanks @jbphet, I clarified the documentation and added a TODO about removing the option once #424 is sorted.

**Comment by jessegreenberg** [comment]
In https://github.com/phetsims/sun/issues/424 @emily-phet said
> Let's wait on this fix for the next round of sims and move forward with RIAW and Friction.

So phetsims/sun#424 is no longer blocking publication of this sim and https://github.com/phetsims/scenery-phet/commit/fa42c378433fdaaec0a46f0c02baaf7ec1be9977 can be brought into the RC branch.

https://github.com/phetsims/scenery-phet/commit/57baa7a2f7f29352f6c8ba381c4093584670baeb could also be included for the RC branch to be closer to master but is only a documentation change.

@jbphet please let me know if I can be of assistance in this.

**Comment by jessegreenberg** [comment]
I realized that this behavior will appear for the NavigationBarSoundToggleButton too because that button also has an alert when pressed. So the above commit will also need to be merged into the release branch.

**Comment by jbphet** [comment]
I've pulled the latest to the 1.6 branch so the fix(es) should be available in v1.6.0-rc.3.

**Comment by KatieWoe** [comment]
Fixed per @lmulhall-phet 1.6.0-rc.3
